<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValueValue</class>
 <widget class="QWidget" name="ValueValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="HSVAdjustmentTool">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HSV Adjustment Tool</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveMatrixButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>511</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Save( Matrix Method)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveLoopButton">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>511</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Save( Loop Method)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CompareButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>511</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Compare Methods</string>
   </property>
  </widget>
  <widget class="QLabel" name="DevedlopedBy">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>560</y>
     <width>346</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Developed By Abdelrahman Mohamed &amp; Ahmed Hassan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UploadButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>12</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>271</y>
     <width>541</width>
     <height>200</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Hue">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="HueValue">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="HueSlider">
      <property name="inputMethodHints">
       <set>ImhDigitsOnly</set>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="orientation">
       <enum>Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Saturation">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Saturation (%)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SaturationValue">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="SaturatedSlider">
      <property name="inputMethodHints">
       <set>ImhDigitsOnly</set>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Value (Gamma)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SaturationValue_2">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="ValueSlider">
      <property name="inputMethodHints">
       <set>ImhDigitsOnly</set>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>400</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>43</y>
     <width>256</width>
     <height>210</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Vertical</enum>
   </property>
   <widget class="QLabel" name="Original">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
    <property name="alignment">
     <set>AlignBottom|AlignHCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView"/>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>43</y>
     <width>256</width>
     <height>210</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Vertical</enum>
   </property>
   <widget class="QLabel" name="Adjusted">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adjusted</string>
    </property>
    <property name="alignment">
     <set>AlignBottom|AlignHCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>graphicsView</tabstop>
  <tabstop>graphicsView_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>